src/legacy/core_plugins/timelion/public/vis/_timelion_editor.scss

```diff
@@ -0,0 +1,31 @@
+.visEditor--timelion{
+
+  vis-options-react-wrapper,
+  .visEditorSidebar__options,
+  .visEditorSidebar__timelionOptions,
+  .visEditor__timelionExpressionInput {
+    @include flex-parent(1, 1, auto, column);
+
+    height: 100%;
+
+    > * {
+      flex-shrink: 1;
+    }
+  }
+}
+
+.visEditor__timelionExpressionInput {
+  .euiFormRow__fieldWrapper {
+    flex-grow: 1;
+  }
+}
+
+.timelionExpressionInput__editor {
+  height: 100%;
+  padding-top: $euiSize;
+
+  @include euiBreakpoint('xs', 's', 'm') {
```

Comment:
I've spent some time to find the root cause of this, but didn't find it..(
It's hard to find it since there are so many angular directives and wrappers. But I merged this with the new React editor, which is waiting for this pr, and there it works pretty good:

![timelion_resize](https://user-images.githubusercontent.com/31325372/71157995-21546e00-2254-11ea-87e6-df209c3d86a1.gif)


Currently the timelion layout is fully adaptive, but have some gaps with responsiveness on small screens (do not work when minimizing the screen, and I don't have any idea why the `ReactResizeDetector` is not working in that case 🤔). 
What do you think if we leave it here as it is, since it will be fixed when the default editor component will be merged?


I've prepared a separate [PR](https://github.com/elastic/kibana/pull/53292) to fix the height of the layout  (seems it was broken unintentionally, but I still want to find the root cause in recent PRs)

Reply:
If the other PR fixes it it's fine for me to leave it like this on this PR, as long as we don't merge this one into `7.x`. Feature freeze isn't far away and I don't want to end up with having one PR merged and other still pending and going into 7.7

Reply:
Resolved